packages/itwin/tree-widget/package.json

```
@@ -33,7 +33,7 @@
     "copy:esm": "cpx \"./src/**/*.{scss,json}\" ./lib/esm",
     "cover": "nyc npm run test",
     "extract-api": "betools extract-api --entry=tree-widget-react",
-    "lint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2",
+    "lint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2 && npx stylelint \"**/*.scss\"",
```

`npx` shouldn't be needed.

```suggestion
    "lint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2 && stylelint \"./src/**/*.scss\"",
```

Maybe separate scripts for `eslint` and `stylelint`?
```
"lint": "npm run lint:eslint && npm run lint:stylelint",
"lint:eslint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2",
"lint:stylelint": "stylelint \"./src/**/*.scss\"",
```